newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -1,92 +1,263 @@
+                    },
+                    {
+                      label: i18n._(t`Open leaderboards`),
+                      click: () => onOpenGameManager('leaderboards'),
+                    },
+                  ]}
+                />
+              </Line>
+              <ResponsiveLineStackLayout noMargin>
+                <GameThumbnail
+                  gameName={game.gameName}
+                  thumbnailUrl={game.thumbnailUrl}
+                />
+                <Spacer />
+                <Column expand justifyContent="space-between">
+                  <ResponsiveLineStackLayout noMargin alignItems="flex-start">
+                    <CardHeader
+                      title={game.gameName}
+                      subheader={
+                        isCurrentGame && (
+                          <Chip
+                            size="small"
+                            label={<Trans>Currently edited</Trans>}
+                            color="primary"
+                          />
+                        )
+                      }
+                    />
+                    <Column expand noMargin>
+                      <ResponsiveLineStackLayout
+                        justifyContent="flex-end"
+                        noColumnMargin
+                      >
+                        <FlatButton
+                          label={<Trans>Access feedback</Trans>}
+                          onClick={() => {}}
+                          disabled={!game.publicWebBuildId}
+                        />
+                        <RaisedButton
+                          label={<Trans>Open in browser</Trans>}
+                          onClick={openGameUrl}
+                          primary
+                          disabled={!game.publicWebBuildId}
+                        />
+                        <IconButton
+                          size="small"
+                          disabled={!game.publicWebBuildId}
+                          onClick={() => setShowShareDialog(true)}
+                          tooltip={t`Share`}
+                        >
+                          <ShareIcon />
+                        </IconButton>
+                      </ResponsiveLineStackLayout>
+                    </Column>
+                  </ResponsiveLineStackLayout>
+                  <ResponsiveLineStackLayout
+                    noMargin
+                    justifyContent="flex-start"
+                  >
+                    <Column expand alignItems="flex-start" noMargin>
+                      <Toggle
+                        labelPosition="left"
+                        onToggle={onToggleDiscoverable}
+                        toggled={getCurrentState(
+                          game.discoverable,
+                          isDiscoverable
+                        )}
+                        label={<Trans>Make discoverable on Liluo</Trans>}
+                        style={{ marginLeft: 0 }}
```

Comment:
We might want to do the reverse:
- Have no margin by default
- Have any code using already `Toggle` to use `Column` instead around the toggle..